guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,56 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {EL} 7 to {EL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
+
+If you have custom Ansible content such as playbooks, job templates inside REX, roles and collections on disk, and you relied on modules being delivered by the Ansible RPM on {Project}, you have to take additional steps to adapt your Ansible installation or migrate your Ansible content.
+
+Ansible Core removed all non-essential modules shipped in the core installation.
+In terms of FQCN notation `namespace.collection.module`, you can continue to use `ansible.builtin.*`, but everything else is missing in Ansible Core.
+That means you will be no longer able to use non-builtin Ansible modules as you were used to and you have to get them from somewhere else, eventually.
+
+Current options to resolve this situation:
+
+* You can obtain additional community-maintained collections that provide the removed functionality from Ansible Galaxy.
+For more information, see https://docs.ansible.com/ansible/devel/user_guide/collections_using.html#installing-collections[Installing collections] in _Ansible Galaxy Documentation_.
```

Comment:
linking to devel is probably good for Foreman "master" branch, but all other branches ideally link to a specific version if possible.

Reply:
`s/devel/latest/` then, it points to "the latest stable"?

Reply:
Or you can explicitly use the core docs for 2.12 at https://docs.ansible.com/ansible-core/2.12/user_guide/collections_using.html

Reply:
Maybe we can use an attribute for this and the set it per Foreman version? Or at least differeniate between versions between EL7 and EL8.

Reply:
EL7 is no more (this doc is explicitly about moving things when coming to EL8), and EL8 is currently 2.12-exclusive, so I don't see a big benefit _right now_.

Reply:
Yes, my bad!

Then "latest" seems to be a good choice!